### DISCUSSION TITLE
Lesson 5 Ganache CLI will not spin up test chain

### DISCUSSION THREAD

**Discussion by anonymouscircle** [smartcontractkit/full-blockchain-solidity-course-py#1815] [General]
Ive seen this is a a common problem but I am still struggling to find a decent solution. 

When using `account = accounts[0]` in deploy.py a cli does not spin up even though my package.json file reads as follows...

`"dependencies": {
    "ganache": "^7.7.2",
    "ganache-cli": "^6.12.2"
  `
I installed using yarn which may be my problem and I am aware that ganache-cli is now ganache but even after doing many work arounds and adding a ganache workspace directly to my brownie networks my ganache test net still cannot be connected to my code and is giving me this error

`HTTPSConnectionPool(host='172.19.112.1', port=7545): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:997)')))`

Wondering if anyone has a way to work around this or a solution. Thinking I might just move on...

**Comment by cromewar** [comment]
Try changing your host, use http:// before the up number or try with '0.0.0.0'

**Reply by cromewar** [reply]
@anonymouscircle does the solution above worked? Also take in mind that is better to install ganache globally.

**Reply by anonymouscircle** [reply]
that didnt work. looking through the documentaion to find a solution. will let you know if i figure anything out

**Reply by anonymouscircle** [reply]
I reinstalled using npm as an admin using `sudo npm install ganache --global` this will solve the problem for users in a WSL environment. Still not sure what to do abt the infura keys...
